content/en/database_monitoring/setup_oracle/troubleshooting.md

```diff
@@ -36,6 +36,17 @@ Ensure that the [recommended Agent version][2] for your hosting type is installe
 ### "Table or view does not exist" error in `agent.log`
 Execute the permission grants listed in the **Grant permissions** step of the [setup instructions][3] for your hosting type.
 
+## Known issues
+
+### No Oracle DB hostname reported
+
+Datadog Agent detects the Oracle DB hostname by running SQL against [V$INSTANCE][4].
```

Comment:
```suggestion
The Datadog Agent detects Oracle DB hostnames by running SQL commands against [V$INSTANCE][4].
```

Reply:
Should `hostnames` and `commands` be singular as the Agent runs only one query which returns only one hostname?

Reply:
Thanks for that question, and apologies for the delay following up here. In this case, no, they should not be singular, as what's being referenced is a general behavior rather than a single occurrence of a behavior. 

If you feel that it's important to emphasize that the Agent runs only one query, which returns only one hostname, then an alternative phrasing would be:
The Datadog Agent detects **the** Oracle DB hostname by running **a** SQL command against [V$INSTANCE]